core/storage/ipfs/merkledag/impl/node_impl.cpp

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "storage/ipfs/merkledag/impl/node_impl.hpp"
+
+#include <libp2p/crypto/sha/sha256.hpp>
+#include <libp2p/multi/content_identifier_codec.hpp>
+#include <libp2p/multi/multibase_codec/codecs/base58.hpp>
+#include <libp2p/multi/multihash.hpp>
+#include "merkledag.pb.h"
```

Comment:
specify full path

Reply:
It's impossible, because this file is generated by protobuf in the temporary build directory during cmake-configure operation.